File: Week 1 (C)/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{   int n;
    do
    {
         n = get_int("Height: ");
    }
    while(n < 1 || n > 8);

    for (int i = 1; i <= n; i++)
    {

        for (int k = n; k > i; k--)
        {
            printf(" ");
        }

        for (int j = i; j > 0; j--)
        {
            printf("#");
        }

        printf(" ");

        for (int j = i; j > 0; j--)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: Week 6 (Python)/sentimental-mario-more/mario.py
from cs50 import *
while True:
    n = get_int("Height: ")
    if n > 0 and n <= 8:
        break

for i in range(n):

    for k in range (n, i + 1, -1):
        print(" ", end="")

    for j in range(i + 1):
        print("#", end="")

    for l in range (i + 1):
        if l == 0:
            print("  #", end="")
        else:
            print("#", end="")
    print()






